WBooks/BookRentals/BookSuggestionsCard/BookSuggestionsCardViewModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BookSuggestionsCardViewModel.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 28/04/2022.
+//
+
+import Foundation
+
+protocol BookSuggestionsCardViewProtocol {
+    func getSuggestions(action: @escaping () -> Void)
+    var suggestions: [Suggestion] { get set }
+}
+
+final class BookSuggestionsCardViewModel: BookSuggestionsCardViewProtocol {
+    var suggestions: [Suggestion] = []
+    private var repository = SuggestionRepository()
+    
+    func getSuggestions(action: @escaping () -> Void) {
+        let onSuccess = { (suggestions: [Suggestion]) in
+            self.suggestions = suggestions
+            self.suggestions.removeAll(where: { suggestion in
+                let link = suggestion.link
+                return link == "https://www.google.com/"
```

Comment:
Link is plenty I think it can be handled like this

`return suggestion.link == "https://www.google.com/"`